scripts/setup_k8s.sh

```diff
@@ -0,0 +1,17 @@
+# !/usr/bin/bash
+
+set -e -x
+
+yarn assets
+# Update filenames of generated CSS to be fingerprinted with matching JS fingerprint
+for file in public/assets/*.js
+do
+  if [[ $file =~ (.*)\.(.*)\. ]]; then
+    if [ -f "${BASH_REMATCH[1]}.css" ]; then
+      mv "${BASH_REMATCH[1]}.css" "${BASH_REMATCH[1]}.${BASH_REMATCH[2]}.css"
+    fi
+  fi
+done
+
+# gzip -S .cgz $(find public/assets -name '*.css')
```

Comment:
I commented this out since it seems like Cloudfront can do this? Or we can leave this in.